modules/odc-creating-helm-charts-using-developer-perspective.adoc

```diff
@@ -27,8 +27,8 @@ image::odc_helm_chart_devcatalog.png[]
 Where available, you can switch between the *YAML View* and *Form View*. The data is persisted when switching between the views.
 ====
 +
-.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Click *Create* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
```

Comment:
If a Helm Chart creates workloads when installed, then the user is redirected to Topology, else is navigated to the Helm release details page.
Workloads are : DaemonSet, CronJob, Pod, Deployment, DeploymentConfig

Reply:
@debsmita1 thank you for the review. I have updated all the suggestions by you. Just want to clarify whether this is an addition to the above step mentioned step. If yes, please let me know.